src/NexusMods.App.UI/Pages/LoadoutPage/LoadoutViewModel.cs

```diff
@@ -199,6 +201,31 @@ public LoadoutViewModel(
                         .ToArray();
 
                     if (ids.Length == 0) return;
+
+                    var dialog = DialogFactory.CreateMessageBox(
```

Comment:
This should be localized. 
@Al12rs @insomnious 

There's a lot of PRs that make it in with hardcoded English; I find myself fixing those down the road.